phpBB/styles/prosilver/template/icons/iconify.html

```diff
@@ -0,0 +1,4 @@
+{% spaceless %}
+	<i class="iconify o-icon {{ ICON ~ (CLASSES ? ' ' ~ CLASSES) }}"{% if S_HIDDEN %}{% if TITLE %} title="{{ lang(TITLE) }}"{% endif %} aria-hidden="true"{% endif %}{{ ATTRIBUTES }}></i>
```

Comment:
hmm now that I see it this will not work this way. It should be a span tag. iconify will actually replace everything with an svg so the next line will be removed. we may also need an inline false flag not sure will have to test it this weekend. Also forgot iconify does not use classess for the icons but instead a data attribute.

`<span class="iconify" title="android" data-icon="mdi-android" data-inline="false"></span>`

lastly since its completely replaced any custom attributes will not work we may need a wrapper around them. 

Let me do some testing...

Reply:
ok so we just need to tweak this line to

`<i class="iconify o-icon {{ CLASSES ? ' ' ~ CLASSES }}" {% if S_HIDDEN %}{% if TITLE %} title="{{ lang(TITLE) }}"{% endif %} aria-hidden="true"{% endif %} data-icon="fa:power-off" data-inline="true" {{ ATTRIBUTES }}></i>`